src/config/Icon.tsx

```diff
@@ -147,6 +147,7 @@ export const Icon = {
   Settings: (props: IconProps) => (
     <NamedIcon name={device.platform === 'ios' ? 'ios-settings' : 'md-settings'} {...props} />
   ),
+  Send: (props: IconProps) => <NamedIcon name={'send'} {...props} />,
```

Comment:
does `send` work for ios and android? if yes, i would suggest to format it `<NamedIcon name="send" {...props} />` because there is no javascript needed for the plain string.

Reply:
checking 2752a877c0fc28a4677f432b87e92ca1689237ee is saying: 
> This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository.

you mean 6d09d940ed07b93248d82864cc5fa3cd0d68ac79?

Reply:
Yes. https://github.com/ikuseiGmbH/smart-village-app-app/commit/6d09d940ed07b93248d82864cc5fa3cd0d68ac79